.circleci/config.yml

```diff
@@ -209,6 +209,27 @@ jobs:
           path: android/app/build/outputs/apk
           destination: apks/release
 
+  # Build Android sourcemap and upload to Instabug
+  instabug-upload-sourcemap-android:
+    <<: *defaults_js
+
+    steps:
+      # Restore workflow workspace
+      - attach_workspace:
+          at: /home/circleci
+
+      - run:
+          name: Build sourcemap
+          command: yarn bundle:android-sourcemap
+      
+      - run:
+          name: Install jq
+          command: apt-get install -y jq
+
+      - run:
+          name: Build sourcemap and upload it to Instabug
+          command: ./scripts/instabug-upload-sourcemap.sh android
```

Comment:
how is the sourcemap linked to the commit number it has been generated from?

Reply:
In the documentation there is nothing about that, so I don't actually know. I suppose Instabug get the app version from both the error reporting and the sourcemap.